.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+const Prettierrc = require("./prettier.config.js");
+
+module.exports = {
+	"env": {
+		"browser": true,
+		"es2021": true
+	},
```

Comment:
Do u know what each line does here?

Reply:
the env browser: creates global  browser variables.
and es2021:  adds all ECMAScript 2021 globals 
says here: https://eslint.org/docs/latest/user-guide/configuring/language-options#specifying-environments

Reply:
> the env browser: creates global browser variables. and es2021: adds all ECMAScript 2021 globals says here: https://eslint.org/docs/latest/user-guide/configuring/language-options#specifying-environments

Ok, why do you need this?

Reply:
Ok, it's fine! I checked the docs 

But I hope you know what every single rule does 😉